Product/Production/Adapters/General/CONNECTAdminGUI/src/main/java/gov/hhs/fha/nhinc/admingui/managed/PatientSearchBean.java

```diff
@@ -638,4 +687,37 @@ public String getDocumentTypeNameFromTheStaticList(String documentType) {
     public String getDocumentInfoModalWindowHeader() {
         return getDocumentTypeName() + " for " + getSelectedCurrentPatient().getName();
     }
+
+    private static UserLogin getCurrentUser() {
```

Comment:
I think we have this method already in https://github.com/CONNECT-Solution/CONNECT/blob/c29ddb2b8842febff6692026126253a99b7e9c9d/Product/Production/Adapters/General/CONNECTAdminGUI/src/main/java/gov/hhs/fha/nhinc/admingui/util/HelperUtil.java#L372